python/lsst/meas/base/plugins.py

```diff
@@ -31,13 +31,48 @@
 import lsst.afw.geom
 
 from .pluginRegistry import register
-from . import baseLib as bl
 from .pluginsBase import BasePlugin
 from .sfm import SingleFramePluginConfig, SingleFramePlugin
 from .forcedMeasurement import ForcedPluginConfig, ForcedPlugin
 from .wrappers import wrapSimpleAlgorithm, wrapTransform
 from .transforms import SimpleCentroidTransform
 
+from . import (
+    ApertureFluxControl,
+    ApertureFluxTransform,
+    BaseTransform,
+    BlendednessAlgorithm,
+    BlendednessControl,
+    CircularApertureFluxAlgorithm,
+    GaussianCentroidAlgorithm,
+    GaussianCentroidControl,
+    GaussianCentroidTransform,
+    GaussianFluxAlgorithm,
+    GaussianFluxControl,
+    GaussianFluxTransform,
+    MeasurementError,
+    NaiveCentroidAlgorithm,
+    NaiveCentroidControl,
+    NaiveCentroidTransform,
+    PeakLikelihoodFluxAlgorithm,
+    PeakLikelihoodFluxControl,
+    PeakLikelihoodFluxTransform,
+    PixelFlagsAlgorithm,
+    PixelFlagsControl,
+    PsfFluxAlgorithm,
+    PsfFluxControl,
+    PsfFluxTransform,
+    ScaledApertureFluxAlgorithm,
+    ScaledApertureFluxControl,
+    ScaledApertureFluxTransform,
+    SdssCentroidAlgorithm,
+    SdssCentroidControl,
+    SdssCentroidTransform,
+    SdssShapeAlgorithm,
+    SdssShapeControl,
+    SdssShapeTransform,
+    )
+
```

Comment:
As unpleasant as it is, these probably also ought to be imported from their subpackages/modules instead of from `.`.